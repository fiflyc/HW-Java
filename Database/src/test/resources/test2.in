1 AAA 000
1 BBB 222
1 AAA 001
1 AAA 003
1 BBB 000
1 AAA 002
1 BBB 111
2 AAA
2 BBB
0