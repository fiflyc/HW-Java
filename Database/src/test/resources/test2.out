000 001 003 002 
222 000 111 
